components/profile_popover/profile_popover.jsx

```diff
@@ -323,6 +324,14 @@ class ProfilePopover extends React.Component {
             );
         }
 
+        if (this.props.user.timezone) {
+            dataContent.push(
+                <span>
+                    <LocalDateTime/>
```

Comment:
Is this supposed to display in the other user's time zone? You're not passing in any props, so it'll just show the current time for the current user

Reply:
🤦‍♂️